packages/@sanity/desk-tool/src/panes/document/statusBar/DocumentStatusBarActions.tsx

```diff
@@ -115,26 +105,16 @@ export function DocumentStatusBarActions(props: {id: string; type: string}) {
       disabled={connectionState !== 'connected'}
     />
   )
-}
+})
 
```

Comment:
Hm, which one?

Reply:
Looks like my IDE failed me here and somehow missed the line number. I think it was about the null guard for editState, but I see it's because of the change that makes `useEditState` never return `null` anymore 👍🏼